### ISSUE TITLE
GSSOC EXT NOTICE: Improper Labeling in the Project

### ISSUE THREAD

**Issue by Meetjain1** [mansiruhil/AmbuFlow#342]
Dear Project Admin @mansiruhil13 

It has come to our attention that your repository not applying the necessary labels to issues and merged PRs, which is affecting contributor scores on the leaderboard.

Please ensure you use the correct labels:

- gssoc-ext
- level1, level2, level3 
- hacktoberfest-accepted (only if your repository have Hacktoberfest topic in description)

*NOTE:* Level labels should not have any dashes between them. Spaces are allowed, but no dashes should be used.

Kindly review and update the labels on your issues and PRs to ensure fairness for all contributors. Failure to address this may result in your project being flagged.

Thank you,
Core Team
GSSoC '24 Extended

**Comment by jyothimanoj12** [comment]
yes i have not been awarded points for solving #164  for the issue #163

**Comment by Meetjain1** [comment]
> yes i have not been awarded points for solving #164 for the issue #163
> 
> also i think this should a level 2 , according to gssoc guidlines, enhanced of an existing feature should be of level 2 and should be awarded 25 points

Thanks for updating @jyothimanoj12 , okay
